### PR TITLE
add podSecurityContext with secure defaults

### PR DESCRIPTION
**Pull request by ckotzbauer**

### CODE REVIEW

**Comment by ckotzbauer** [comment]
@bzon Is there anything missing to get this merged?

File: chart/prometheus-msteams/README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Otherwise you can also set the value by specifying the template data directly vi
 | `tolerations`                              | Pod tolerations                                                                                                                                               | `{}`                                            |
 | `priorityClassName`                        | Pod priority class                                                                                                                                                     | `""`
 | `podAnnotations`                           | Pod annotations                                                                                                                                               | `{}`                                            |
+| `podSecurityContext`                       | Pod securityContext                                                                                                                                           | See [default](./prometheus-msteams/values.yaml) |
```
[the remark this answers is not in the document]
Comment (ckotzbauer) [reply chart/prometheus-msteams/README.md:110]: This is the same path as already present by "resources"...

**Comment by bzon** [comment]
@ckotzbauer apologies it took time to review this. Thanks for your contribution!

**Comment by ckotzbauer** [comment]
@bzon Could you please release a new version of the chart?

**Comment by bzon** [comment]
@ckotzbauer oh yeah! I just released it now.

**Pull request merged**